csp_billing_adapter/csp_cache.py

```diff
@@ -110,25 +110,25 @@ def add_usage_record(record: dict, cache: dict) -> None:
 
 def cache_meter_record(
     cache: dict,
-    record_id: str,
+    record_ids: list,
```

Comment:
This should now be a dict.

```suggestion
    record_ids: dict,
```